.github/renovate.json

```
@@ -73,7 +73,8 @@
         "api/**",
         "catalog/**",
         "ingestion_server/**",
-        "documentation/**"
+        "documentation/**",
+        "py_packages/openverse-attribution/**"
```

This is 100% a complete and total nit and personal preference but the asymmetry of `py_packages/openverse-attribution` (and similarly `js_packages/eslint-plugin`) really bugs me (for no good technical reason I could possible reasonably argue for, I acknowledge LOL!)

```suggestion
        "py-packages/openverse-attribution/**"
```

---

Oooh good eye. It's definitely not a personal preference, a mix of underscores and hyphens reads quite badly. Now that you've pointed it out, I would also be completely bothered by it all the time.